### DISCUSSION TITLE
I don't know how to use certain items.

### DISCUSSION THREAD

**Discussion by KaziPlays** [BEEmod/BEE2-items#4194] [Tech Support / Q&A]
I'm having problems figuring out how to use certain items, so I'm gonna ask for some help, here is the list of things I want to do:

1-Other types of doors;

2-Co-op Player droppers;

3-Chamberlock and Co-op checkpoints;

4-Vactube joints;

5-Portal 1 Unstationary scaffold items.

Please give some advice & help.

**Comment by saismeesaimonsaimes** [comment]
You should use the [Discord](https://discord.gg/hnGFJrz) for general questions. The GitHub is only for bugs and feature requests.
